app/config/FrontendAppConfig.scala

```diff
@@ -63,8 +63,8 @@ class FrontendAppConfig @Inject() (configuration: Configuration) {
   lazy val languageTranslationEnabled: Boolean =
     configuration.get[Boolean]("microservice.services.features.welsh-translation")
 
-  lazy val countdownLength: String = configuration.get[String]("timeout.countdown")
-  lazy val timeoutLength: String = configuration.get[String]("timeout.length")
+  lazy val countdownLength: Int = configuration.get[Int]("timeout.countdown")
+  lazy val timeoutLength: Int = configuration.get[Int]("timeout.length")
 
   private def getInt(path: String): Int = configuration.get[Int](path)
 
```

Comment:
I think you've missed the contact-frontend changes for the Deskpro link here?

https://github.com/hmrc/register-trust-other-individual-frontend/blob/master/Migration%20to%20GovUk%20Frontend.md#deskpro-link

Reply:
That should be fixed now